cryptoconditions/types/base_sha256.py

```diff
@@ -1,26 +1,19 @@
-from abc import abstractmethod
+import hashlib
 
 from cryptoconditions.fulfillment import Fulfillment
-from cryptoconditions.lib.hasher import Hasher
 
 
-class BaseSha256Fulfillment(Fulfillment):
+class BaseSha256(Fulfillment):
```

Comment:
Hmmm, good point. I think the idea is that the `Fulfillment` is "SHA" agnostic. This way, one can imagine in the near future a new suite of types based of SHA-512 for instance, that would still use the class `Fulfillment` as a parent class.

_As a side note_: These things could be achieved via mixins probably.